src/palette/internal/palettecreator.cpp

```diff
@@ -231,7 +231,6 @@ PalettePtr PaletteCreator::newKeySigPalette()
 {
     PalettePtr sp = std::make_shared<Palette>(Palette::Type::KeySig);
     sp->setName(QT_TRANSLATE_NOOP("palette", "Key signatures"));
-    sp->setMag(0.8);
```

Comment:
We're handling element-scaling on a per-cell basis now, rather than a per-palette basis.

The user can still set a custom magnification on a palette, if, say, they want the whole "Articulations" palette to be scaled up. This composes nicely with the per-cell element scaling.